src/odemis/dataio/test/multi_tiff_test.py

```diff
@@ -0,0 +1,98 @@
+
+import datetime
+import glob
+import json
+import logging 
+import os
+import unittest
+
+import numpy
+
+from odemis import model
+from odemis.dataio import tiff
+from odemis.util.dataio import open_acquisition
+
+logging.basicConfig(level=logging.DEBUG)
+
+def write_test_metadata(path: str):
+    """Write test metadata to json file.
+    :param path: (str) path to test data (directory of *.tif images)"""
+
+    filenames =  glob.glob(os.path.join(path, "**/*.tif*"), recursive=True) 
+    test_metadata = {}
+
+    for fname in filenames:
+
+        try:
+            data = open_acquisition(fname)[0].getData()
+
+            # format metadata for tests
+            tmd = data.metadata
+            tmd["filename"] = os.path.basename(fname).replace(".tif", "")
+            tmd["shape"] = data.shape
+            tmd["dtype"] = str(data.dtype)
+            tmd["length"] = 1
+
+            # convert timestamp into datetime
+            ts = data.metadata[model.MD_ACQ_DATE]
+            tmd["timestamp"] = ts
+            tmd["datetime"] = datetime.datetime.fromtimestamp(ts).strftime('%Y-%m-%d %H:%M:%S')
+
+            test_metadata[tmd["filename"]] = tmd
+
+        except Exception as e:
+            logging.warning(f"Could not parse {fname}. Exception: {e}")
+
+    # save test_metadata as json
+    with open(os.path.join(path, "test_metadata.json"), "w") as f:
+        json.dump(test_metadata, f)
+    
+    return test_metadata
+
+
+class TestMultiTiffIO(unittest.TestCase):
+
+  def test_manufacturer_metadata_parsers(self):
+    """Test open_data function with multiple manufacturers and metadata parsers."""
+
+    # test data path (only available on hosted runner atm)
+    TEST_PATH = os.path.join(os.path.expanduser('~'), "test_data/meteor/test_images")
```

Comment:
I'm confused. Where is this directory? In Google Drive, the closest I can see is: Software Engineering/Test data/meteor-examples/

Reply:
It's on the self hosted server